src/en/news/blog/2011/hiring/index.md

```diff
@@ -0,0 +1,10 @@
+---
+title: "Hiring!"
```

Comment:
Is this article still needed ?

Reply:
i think it makes sense to keep these for historical completeness.